.test.output

```diff
@@ -0,0 +1 @@
+cargo nextest run --workspace --no-fail-fast --no-default-features --features "default" 
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏. 👀